### ISSUE TITLE
Enhance Presentation server to use valid Coco Pharmaceutical userids

### ISSUE THREAD

**Issue by davidradl** [odpi/egeria#3450]

**Comment by mandy-chessell** [comment]
Thanks @davidradl :)

**Comment by davidradl** [comment]
@mandy-chessell There seems to be some inconsistencies with the user definitions. I had copied a subset of the UI chassis UI users into the presentation server users. I was looking at Faith Broker's userid in the Egeria code base and see: 

* `open-metadata-resources/open-metadata-deployment/sample-data/coco-pharmaceuticals/vars/users.yml`
Has username: fbroker

* /Users/davidradley/egeria/open-metadata-resources/open-metadata-samples/open-metadata-security-samples/src/main/java/org/odpi/openmetadata/metadatasecurity/samples/CocoPharmaServerSecurityConnector.java
Has final String faithBrokerUserId    = "faithbroker";

* src/main/java/org/odpi/openmetadata/userinterface/uichassis/springboot/auth/db/DemoInMemoryInitializingBean.java
addUser("faith","Faith Broker ", "admin","/resources/img/faith.png",Role.DATA_ANALYST);
addUser("faithbroker", "Faith Broker", "admin", "/resources/img/user.png" );

* `org/odpi/openmetadata/accessservices/governanceprogram/samples/leadership/GovernanceLeadershipSample.java`
private static final String faithBrokerUserId  = "faithBroker";

* `presentation-server/nodejs/src/server/db/users.js` Has a subset of the UI chassis users  
addUser("faith","Faith Broker ", "admin","/resources/img/faith.png",Role.DATA_ANALYST);
addUser("faithbroker", "Faith Broker", "admin", "/resources/img/user.png" );

I assume that the userids exposed in the UIs are not the valid ones. Is the users.yml list the definitive user definitions? Or is UI Chassis user list the definitive ones and the short versions of the user is e.g. faith should be removed?

At the moment Faith's userid is defined as fbroker, faith, faithbroker or faithBroker. 

I was hoping to fix the presentation server part of this in this issue as I can test that and raise issues against the other areas as appropriate.

fyi @planetf1.

**Comment by planetf1** [comment]
I would suggest fixing all the inconsistencies in the same PR

IMO 'faithbroker' is correct - - I seem to remember when I added to the UI I left the existing fbroker in place (in case ING were using it) . I would suggest copying in bogdan/chris to check no undue impact from the other changes

**Comment by planetf1** [comment]
Also to be made consistent - the ldap definitions in ./open-metadata-resources/open-metadata-deployment/sample-data/coco-pharmaceuticals/ldap which use another form for example:
```
 dn: uid=Faith_Broker,ou=People,dc=example,dc=org
 objectclass: top
 objectclass: person
 objectclass: organizationalPerson
 objectclass: inetOrgPerson
 cn: Faith Broker
 gn: Faith
 sn: Broker
 uid: Faith_Broker
 userPassword: password
```

**Comment by davidradl** [comment]
@planetf1 . So your suggestion is that list in src/main/java/org/odpi/openmetadata/userinterface/uichassis/springboot/auth/db/DemoInMemoryInitializingBean.java
is accurate? 

Apart from we need to remove 
` addUser("faith","Faith Broker ", "admin","/resources/img/faith.png",Role.DATA_ANALYST);
        addUser("callie","Callie Quartile ", "admin","/resources/img/callie.png",Role.HR);
        addUser("zach", "Zach Now", "admin", "/resources/img/user.png" );
        addUser("steves", "Steve Starter", "admin", "/resources/img/user.png" );
        addUser("terri", "Terri Daring", "admin", "/resources/img/user.png" );`
I notice the list has 
        `addUser("designa", "designa", "admin", "/resources/img/user.png" );`
which I am not sure what the long named version is
and 
        `addUser("stewFaster", "stewFaster", "admin", "/resources/img/user.png" );`
this has capitalization which does not match the rest of the userids.

**Comment by davidradl** [comment]
@cmgrote and @bogdan-sava we are looking to standardize the coco users in the code base. If we standardize on something based on the y last update - would there be implications for you?

**Comment by planetf1** [comment]
I attempted to make that file accurate when I last edited it but that was probably 1.5 years ago, so ids may have been changed since. The reason for opening #1140 was to get things consolidated - though there have been changes since then.

I think the most important aspect is consistency. Beyond that I would go with
 - all lower case userids
 - generally of form <forename><surname> like faithbroker
 - using only the set defined in the coco pharma persona documentation

The security plugin will also refer to explicit users so needs to form part of the evaluation too - this can be found at 9:56]
/Users/jonesn/src/egeria/open-metadata-resources/open-metadata-samples/open-metadata-security-samples/src/main/java/org/odpi/openmetadata/metadatasecurity/samples CocoPharmaServerSecurityConnector.java - interestingly this does use 'stewFaster'

I think any reasonable proposal is ok as long as it aligns all these pieces - our notebooks, security plugin, UI #1, UI #2, documentation/readmes, any presentations, ldap definitions. The most likely correct sources are probably the security plugin tbh as they follow the coco pharma persona docs very closely

Also I noticed open-metadata-implementation/adapters/open-connectors/data-store-connectors/file-connectors/csv-file-connector/src/test/resources/SimpleColumnsWithColumnNames.csv uses the same set.
